app/views/community/brief_introductions/index.html.haml

```diff
@@ -14,7 +14,7 @@
 
       .flex-shrink-0{ class: "lg:w-[400px] w-100" }
         .c-youtube-container
-          %iframe{ width: "560", height: "315", src: "https://www.youtube-nocookie.com/embed/zoGruqFThFE", title: "YouTube video player", frameborder: "0", allow: "accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture", allowfullscreen: true }
+          %iframe{ width: "560", height: "315", src: "https://www.youtube-nocookie.com/embed/xVl1NZaJYIg", title: "YouTube video player", frameborder: "0", allow: "accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture", allowfullscreen: true }
```

Comment:
I think this is wrong.
I think the ID should be added to the controller (`app/controllers/community/brief_introductions_controller.rb`) and this should be changed to `@ids.first`

Reply:
Good idea. Updated.